docs/devel/report.md

```diff
@@ -2,12 +2,11 @@
 
```

Comment:
The PR #113 and changes https://github.com/redhat-openshift-ecosystem/opct/pull/113/files#diff-85243efb2cae91b8faf70fab931fa35f1f8511d51610860985b6e2408b1c0e2b must be incorporated somewhere in this PR or isolated one.